tools_docker/README.md

```diff
@@ -146,13 +145,57 @@ Exemple to use Tracim with ElasticSearch-ingest: (you need to create your elasti
 
 ⚠ After execute one of these command, Tracim will be reachable on your system on port 8080.
 
+#### Running with gocryptfs encryption
+
+Exemple with basic instance of Tracim (local usage with webdav and caldav) with encrypted storage:
+
+for this exemple,
+you first need to write password you want in `~/tracim/secret/password.txt` file.
+
+```bash
+echo "password" -> ~/tracim/secret/password.txt
+```
+
+```bash
+docker run \
+       -it \
```

Comment:
is `-it` wanted here?

Reply:
Probably not, need to be tested with and without to verify